<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExportDialog</class>
 <widget class="QDialog" name="ExportDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>446</width>
    <height>244</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>446</width>
    <height>244</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Exporter les diapositives</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="formatLabel">
     <property name="text">
      <string>Format de destination :</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="formatComboBox"/>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="selectionLabel">
     <property name="text">
      <string>Sélection des diapositives :</string>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QRadioButton" name="allSlidesRadioButton">
     <property name="text">
      <string>Toutes les diapositives</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QRadioButton" name="rangeRadioButton">
       <property name="text">
        <string>Diapositves de</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="fromSpinBox">
       <property name="minimum">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="toLabel">
       <property name="text">
        <string>à</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="toSpinBox">
       <property name="minimum">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="9" column="1">
    <widget class="QRadioButton" name="currentSlideRadioButton">
     <property name="text">
      <string>Diapositive courante</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="directoryLabel">
     <property name="text">
      <string>Dossier de destination :</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLineEdit" name="directoryLineEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="placeholderText">
        <string>Sélectionnez un répertoire...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset theme="folder">
         <normaloff/>
        </iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="templateLabel">
     <property name="text">
      <string>Modèle des fichiers :</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="templateLineEdit">
     <property name="toolTip">
      <string>&lt;strong&gt;Tokens disponibles :&lt;/strong&gt;
&lt;ul&gt;
&lt;li&gt;&lt;strong&gt;%n&lt;/strong&gt; : Nom de la diapositive&lt;/li&gt;
&lt;li&gt;&lt;strong&gt;%i&lt;/strong&gt; : Numéro de la diapositive&lt;/li&gt;
&lt;li&gt;&lt;strong&gt;%s&lt;/strong&gt; : Nom du diaporama&lt;/li&gt;
&lt;li&gt;&lt;strong&gt;%f&lt;/strong&gt; : Format de destination&lt;/li&gt;
&lt;/ul&gt;</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="qualityLabel">
     <property name="text">
      <string>Qualité :</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QSlider" name="qualitySlider">
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="singleStep">
        <number>10</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="qualitySpinBox">
       <property name="suffix">
        <string>%</string>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="singleStep">
        <number>10</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ExportDialog</receiver>
   <slot>validate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>397</x>
     <y>221</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>243</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ExportDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>441</x>
     <y>227</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>243</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>templateLineEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>ExportDialog</receiver>
   <slot>enableOkButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>310</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>222</x>
     <y>104</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qualitySlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>qualitySpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>280</x>
     <y>40</y>
    </hint>
    <hint type="destinationlabel">
     <x>425</x>
     <y>42</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qualitySpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>qualitySlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>418</x>
     <y>42</y>
    </hint>
    <hint type="destinationlabel">
     <x>238</x>
     <y>45</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>enableOkButton()</slot>
  <slot>validate()</slot>
 </slots>
</ui>
